site/content/guides/claude-code-monitoring.md

```diff
@@ -0,0 +1,415 @@
+  activated: true,
+  // Whether alerts are suppressed (false = alerts will be sent)
+  muted: false,
+  // Whether the check is expected to fail (false = failure will trigger alerts)
+  shouldFail: false,
+  // Geographic regions where the check will run
+  locations: [
+    'us-east-1', // US East (N. Virginia)
+    'eu-west-1', // EU West (Ireland)
+  ],
+  // How often the check runs in minutes
+  frequency: 30,
+  // Alert escalation policy defining when and how alerts are sent
+  alertEscalationPolicy: AlertEscalationBuilder.runBasedEscalation(1, {
+    // Number of failed runs before sending alert (0 = immediate)
+    amount: 2,
+    // Time interval between escalation checks in minutes
+    interval: 5,
+  }, {
+    // Whether percentage-based escalation is enabled
+    enabled: false,
+    // Percentage threshold for escalation (when enabled)
+    percentage: 10,
+  }),
+  // Retry strategy configuration for failed checks
+  retryStrategy: RetryStrategyBuilder.linearStrategy({
+    // Initial wait time before first retry in seconds
+    baseBackoffSeconds: 30,
+    // Maximum number of retry attempts
+    maxRetries: 2,
+    // Maximum total time for all retries in seconds
+    maxDurationSeconds: 300,
+    // Whether retries should use the same region as original check
+    sameRegion: true,
+  }),
+  // Whether to run checks in multiple locations simultaneously
+  runParallel: true,
+})
+```
+
+These settings are all optional, and any not specified will default back to the account or project default, but it's good to include them, especially the logical ID, which will make it easy to update this check with later deployments.
+
+Later in this tutorial we'll check the consistency of this config file with the `deploy` command. For now you can move on to creating more checks.
+### 2. Add our new Check to Claude's Context
+
+With this check created, add their references to Claude's context by giving Claude Code the prompt:
+
+```{title="Prompt:"}
+update my CLAUDE.md file with the new code in /__checks__
+```
+
+Let's take one additional step, add the following lines to your updated CLAUDE.md file:
+
+```markdown
+- When writing Playwright, don't set locators equal to const, rather just perform expect tests directly on locators
+- When writing a spec.ts file, don't use locators based on CSS class
+```
+
+Pro tip: from the Claude Code prompt, just hit octothorp "#" to add to Claude's memory.
+
+### 3. Use Claude Code to create multiple Browser Checks
+
+Now that we've got a working test and, let's let Claude Code create another check for us:
+
+```{title="Prompt:"}
+Create a new browser check for https://danube-web.shop/ that clicks on a single button then checks the visibility of 10 different elements on the page
+```
+
+In order to scan the URL provided, Claude Code will confirm that it can use the Playwright MCP to open a browser and access the site. The result is a nice clean test written to the spec:
```

Comment:
Will it do that even though Playwright MCP should have been used with the earlier example?